symfony/test/functional/cypress/integration/orangehrm/admin/education.spec.js

```diff
@@ -0,0 +1,180 @@
+import user from '../../../fixtures/user.json'
+import promisify from 'cypress-promise'
+
+
+
+//Get the current education list count
+const getEducationListCount = async() =>{ 
+    let txt = await promisify(cy.get('.oxd-text').contains('Found').invoke('text'))
+    return parseInt(txt.split(' ')[0])
+}
+
+describe('Qualification - Education', function () {
+
+    const characterLengthof51 = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
```

Comment:
better to define more than 50,100,200 chars in fixtures files

Reply:
I have defined different length chars in a fixtures file